content/api/suppression-list.apib

```diff
@@ -4,48 +4,84 @@ description: Manage your suppression list - a list of recipient email addresses
-| description | string | Short explanation of the suppression | no | |
+Each subaccount has it's own independent suppression list.
+Set the `X-MSYS-SUBACOOUNT` header to a subaccount's ID to search and operate on that subuaccount's suppresion list.
+By default, the search endpoint works on all lists in the account (Master account and subaccount lists).
 
+## Suppression Object
 
-## Bulk Insert/Update [/suppression-list/]
++ Data Structure: Attributes
+    + recipient (string) - Email address to be suppressed
+    + type (enum) - Type of suppression record
+        + transactional
+        + non_transactional
+    + source (enum) - Source responsible for inserting the list entry.
+        + Spam Complaint
+        + List Unsubscribe
+        + Bounce Rule
+        + Unsubscribe Link
+        + Manually Added
+        + Compliance
+    + description (string) - Explanation for the suppression
+    + created (string) - Date suppression was created
+    + updated (string) - Last time the suppression was updated
+    + transactional (boolean) - Whether the recipient requested to not receive any transactional messages. Deprecated in favor of `type`
+    + non_transactional (boolean) - Whether the recipient requested to not receive any non-transactional messages. Deprecated in favor of `type`
+    + subaccount_id (number) - Which subaccount the recipient is suppressed for, only returned if suppressed for a specific subaccount.
 
-### Insert or Update List Entries [PUT]
++ Sample
+    ```
+    {
+        "recipient": "recip@example.com",
+        "type": "non_transactional",
+        "source": "List Unsubscribe",
+        "description": "Unsubscribed using list unsubscribe header",
+        "created" : "2017-10-01T12:00:00+00:00",
+        "updated" : "2017-10-01T12:00:00+00:00",
+        "transactional": false,
+        "non_transactional" true
+    }
+    ```
 
-Bulk insert or update entries in the suppression list by providing a JSON object, with a `recipients` key containing an array of recipients to insert or update, as the PUT request body. Maximum size of the JSON object is 50mb. Maximum recipients at a time is 10,000. At a minimum, each recipient must have a valid email address and a suppression type: `transactional` or `non_transactional`. The optional `description` key can be used to include an explanation of what type of message should be suppressed.
+## Insert & Update [PUT /suppression-list/]
 
-If the recipient was added to the list by Compliance, it cannot be updated.
+Bulk insert or update entries in the suppression list.
 
-Please note that in the unlikely scenario where your receive a HTTP 5xx level error response while bulk loading, that some of your suppression entries may have been successfully inserted or updated. If this occurs, please re-submit your original request again for processing.
+If a recipient was added to the list by compliance, it cannot be updated.
```

Comment:
Outside the scope of this PR, but "by compliance" feels weird. Even saying "by our compliance system" or something like that helps indicate it is our system doing it.